src/pages/api/contactFooter.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
```

Comment:
@Cameron-Porter  is there a way we could reuse the existing contact form api vs creating a new route for this? I understand there are different requirements for each form but seems like something we should be able manage in one route. Thoughts?

Reply:
I could try and implement the additions within the api/contact.js file if that is what you mean. Otherwise, I could create a new test file for the separate api file.

Reply:
Yeah I think reuse of the current api rout and updating existing tests to cover the changes would acceptable.

Reply:
Ok, I will give that a shot and send it back to see if it works. Great learning opp!

Reply:
@Cameron-Porter I think I am seeing what @andrew-t-james, lets look in to refactoring `contact.js` in lieu of adding this additional file.  We want to aim for DRY ( Dont Repeat Yourself ) on the code.  So much of this duplicates efforts in the `contact.js` file.